pkg/helm/templates_test.go

```diff
@@ -0,0 +1,57 @@
+package helm
+
+import (
+	_ "embed"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testdata/spec-before.yaml
+var specBeforeTemplating string
+
+//go:embed testdata/spec-after.yaml
+var specAfterTemplating string
+
+//go:embed testdata/values.yaml
+var valuesData string
+
+func Test_ApplyTemplates(t *testing.T) {
```

Comment:
let's expand this test to cover malformed templates, and missing values in values.yaml (because those are the ones we'll see in production the most)

Reply:
updated